# Description: use the same makefile targets as on other platform
# Author: Damyan Ivanov <dmn@debian.org>
# Forwarded: no
--- a/builds/posix/prefix.freebsd
+++ b/builds/posix/prefix.freebsd
@@ -22,7 +22,7 @@ OS_ServerFiles=inet_server.cpp
 
 PROD_FLAGS=-O -fno-builtin -DNDEBUG -DFREEBSD -pipe -MMD -fPIC
 DEV_FLAGS=-ggdb -DFREEBSD -pipe -MMD -p -fPIC -Wall
-EMBED_UTIL_TARGETS=gstat gsec nbackup fbguard fb_lock_print
-CLIENT_UTIL_TARGETS=gstat gsec fbguard fbmgr_bin nbackup fb_lock_print
+EMBED_UTIL_TARGETS=gstat gsec fbguard nbackup fb_lock_print fbsvcmgr fbtracemgr
+CLIENT_UTIL_TARGETS=gstat gsec fbguard fbmgr_bin nbackup fb_lock_print fbsvcmgr fbtracemgr
 
 Physical_IO_Module=os/posix/unix.cpp
